src/components/slice-label.jsx

```diff
@@ -34,15 +40,16 @@ export default class SliceLabel extends React.Component {
       assign({padding: 0}, props.style),
       props.slice.data
     );
+    const events = Helpers.getPartialEvents(props.events, props.index, props);
     return (
       <VictoryLabel
         x={position[0]}
         y={position[1]}
         data={props.slice.data}
         style={style}
-      >
-        {label}
-      </VictoryLabel>
+        text={label}
```

Comment:
Ran into the same thing here: https://github.com/FormidableLabs/victory-pie/pull/63/files#diff-b90949fbe161cdf271232a9b991d3c46R44